testsuite/lb_cpu.py

```diff
@@ -0,0 +1,24 @@
+
+# Basic tests of the Lattice Boltzmann implementation
```

Comment:
Please remove the documentation here since this information is already given in the common file and we don't want to maintain two places for the test documentation.